estimagic/optimization/default_algo_options.py

```diff
@@ -0,0 +1,27 @@
+# Inspired by scipy L-BFGS-B defaults, but rounded
+RELATIVE_CRITERION_TOLERANCE = 2e-9
+
+# Disabled by default because it is very problem specific
+ABSOLUTE_CRITERION_TOLERANCE = 0
+
+# Same as scipy
+GRADIENT_TOLERANCE = 1e-5
```

Comment:
Pounders has an three gradient convergence criteria (p. 17 http://web.mit.edu/tao-petsc_v3.7/tao_manual.pdf). Change names to `(GRADIENT|CRITERION)_(ABSOLUTE|RELATIVE|...)_TOLERANCE`.

Reply:
Done